### ISSUE TITLE
Comment out and replace any small http requests

### ISSUE THREAD

**Issue by eblur** [astropy/astropy-tutorials#348]
e.g., SkyCoords from_name calls Sasame. Frequently breaks our testing infrastructure.

@adrn

**Comment by kakirastern** [comment]
Yeah, I am fixing them in PR #341. Let me work on the PR some more before our meeting this coming Tuesday so they can be merged... Sorry about the delay @eblur!

**Comment by kakirastern** [comment]
I will help @adrn to work on it... since I have already started

**Comment by kakirastern** [comment]
Think the bulk of the work has been done in #341... But remember to rebase after every merge, I noticed that some notebook revert back to an older state after merge without rebase, such as the "units_and_integration" tutorial. 😉

The error was probably due to the spacing in "HCG 7". After you've changed that to something like "hcg7" the Circle CI error goes away. So, it is more like some syntax issue.

**Comment by kakirastern** [comment]
Wait... why are the latest merges not showing up in "learn.astropy.org"? We might have another issue to deal with here. I just checked. Something isn't quite right. @adrn Can you confirm?

**Comment by kakirastern** [comment]
I am guessing the public URL keeps changing as we merge pull requests, so we might need to update that manually regularly, or find means to automate the process. I have been using this [https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html) as a reference...

**Comment by adrn** [comment]
@kakirastern Please try to keep discussion in an issue relevant to that issue - if you want to discuss something else, please just open a new issue. The circleCI build for master (that generates the live version of the site at learn.astropy.org) failed because of a transient issue (`skyserver.sdss.org` was down and one of the tutorials does a web request). But that failure just bolsters this particular issue...

I'm working on this locally, so I'll have a PR in soon.

**Comment by kakirastern** [comment]
@adrn Yup, no problem... I will leave it for you to fix. 😉

**Comment by kakirastern** [comment]
Sorry about the confusion earlier @adrn.

**Comment by adrn** [comment]
No worries!
